charts/base/templates/_env.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "base.findEnvValue" -}}
+  {{- $EnvVarName := .EnvVarName -}}
+  {{- range .Envs -}}
+    {{- if eq .name $EnvVarName -}}
+      {{ .value }}
+    {{- end }}
+  {{- end }}
+{{- end }}
```

Comment:
Great idea to put this into the base chart! I just added the required release of the base chart and did some code styling adjustments. 😊